modules/nixos/hydra.nix

```diff
@@ -51,6 +51,7 @@ in
         printf "$machines" > $out
         substituteInPlace $out --replace-fail 'ssh-ng://' 'ssh://'
         substituteInPlace $out --replace-fail ' 80 ' ' 3 '
+        substituteInPlace $out --replace-fail ' 14 ' ' 2 '
```

Comment:
This should change depending on https://github.com/nix-community/infra/pull/1807/files#r2075684863, correct?